integration/mediation-tests/tests-mediator-1/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/enrichAddBodyToChildTestProxy.xml

```diff
@@ -12,7 +12,7 @@
             <log level="full" />
             <enrich>
                 <source type="body" clone="true"/>
-                <target xmlns:ns="http://services.samples" action="child" xpath="//ns:getQuoteResponse/ns:return" />
+                <target type="custom" xmlns:ns="http://services.samples" action="child" xpath="//ns:getQuoteResponse/ns:return" />
```

Comment:
We have added "custom" as the type for all sources and targets, is that correct?

Reply:
Yes, custom type only we give xpath. And before my fix on enrich mediators it can accept the source and target without specifying type when the type is only for custom.